### PR TITLE
openssl@1.1 1.1.1v

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/openssl@1.1.rb [hunk 1]
```diff
@@ -35,21 +35,21 @@ class OpensslAT11 < Formula
...
     end
 
     resource "ExtUtils::MakeMaker" do
-      url "https://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.48.tar.gz"
-      mirror "http://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.48.tar.gz"
-      sha256 "94e64a630fc37e80c0ca02480dccfa5f2f4ca4b0dd4eeecc1d65acd321c68289"
+      url "https://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.70.tar.gz"
+      mirror "http://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.70.tar.gz"
+      sha256 "f108bd46420d2f00d242825f865b0f68851084924924f92261d684c49e3e7a74"
     end
```
Comment (carlocab) [inline Formula/openssl@1.1.rb:53]: I wonder if these resources are even needed at runtime. They look build-only to me.

**Review by carlocab — APPROVED** [review]

File: Formula/openssl@1.1.rb (hunk 1 above)
Comment (ZhongRuoyu) [reply Formula/openssl@1.1.rb:53]: Looks that way.

Comment (carlocab) [reply Formula/openssl@1.1.rb:53]: Same thing for the other `openssl@*` formulae.

**Review by carlocab — APPROVED** [review]

File: Formula/openssl@1.1.rb (hunk 1 above)
Comment (ZhongRuoyu) [reply Formula/openssl@1.1.rb:53]: Ok, error is unrelated. Btw, any idea what's going on with `dotnet` linkage on Linux @carlocab?

**Comment by carlocab** [comment]
`dotnet` linkage failure happens on `master` too. It expects to link to `openssl@1.1` via `$HOMEBREW_PREFIX/lib`.

File: Formula/openssl@1.1.rb (hunk 1 above)
Comment (ZhongRuoyu) [reply Formula/openssl@1.1.rb:53]: > Same thing for the other openssl@* formulae.

- #138320
- #138322

**Pull request merged**